app/utils/compare-files.ts

```diff
@@ -40,15 +50,14 @@ const sortByFile2Analyses = (
   const analysis1ComputedRisk = Number(a?.analysis2?.computedRisk);
   const analysis2ComputedRisk = Number(b?.analysis2?.computedRisk);
 
-  if (analysis2ComputedRisk < analysis1ComputedRisk) {
+  if (isNaN(analysis1ComputedRisk)) {
     return -1;
   }
 
-  if (analysis2ComputedRisk > analysis1ComputedRisk) {
-    return 1;
-  }
-
-  return 0;
+  return (
+    Number(sortPriorityMap[analysis2ComputedRisk]) -
```

Comment:
do we require number conversion since it is going to return a number

Reply:
Yeah. Typescript safety check. Since `sortPriorityMap` is an object with keys as type `number` typescript thinks that that expression could return undefined. Even though the keys would always fall within the numbers predefined in the `sortPriorityMap` object.